docs/index.html

```diff
@@ -18,30 +18,34 @@
   <body>
     <section>
       <h1>What is a 1x Engineer?</h1>
-      <p>You might have already heard of a 10x engineer. Probably too often, actually. If there's such a thing as a 10x
+      <p>You might have already heard of a <a href="http://10x.engineer/">10x engineer</a>. Probably too often, actually. If there's such a thing as a 10x
         engineer, surely there must be a 1x engineer, too?</p>
       <p>Of course there is! Let's dig into a non-exhaustive list of what qualities make up a 1x engineer.</p>
     </section>
     <section class="nes-container with-title">
       <h2 class="title">A 1x Engineer...</h2>
       <ul class="lists">
         <li>Searches Google when they're not sure what's up.</li>
+        <li>Makes it all up as they go.</li>
         <li>Spends time on things outside of engineering, like hobbies, friends, and family.</li>
         <li>Writes code that &emdash; <em>gasp</em> &emdash; has bugs</li>
-        <li>Writes code that others can read</li>
-        <li>Is a team player that goes to the same meetings their co-workers are required to go to</li>
+        <li>Writes code that others can read and easily maintain.</li>
+        <li>Is a team player that goes to the same meetings their co-workers are required to go to.</li>
         <li>Reads the Docs.</li>
         <li>Updates the Docs.</li>
         <li>Doesn't need to be passionate about the code they write or the problems they solve, but may be.</li>
         <li>Are willing and able to collaborate with others.</li>
         <li>Willing to admit when they're wrong, and aren't afraid to say "I don't know"</li>
-        <li>Will help up-level others.</li>
-        <li>Copy/pastes code snippets from StackOverflow.</li>
+        <li>Will help up-level others and enjoys mentoring.</li>
+        <li>Copy/pastes code snippets from Stack Overflow and in some cases even their own previous answers.</li>
         <li>Asks questions.</li>
         <li>Can be thankful for others' time, effort, and energy.</li>
         <li>Appreciates code reviews from their peers that are constructive and helpful.</li>
         <li>Is open to learning.</li>
         <li>Asks for help when they need it.</li>
+        <li>Is not a douchebag.</li>
+        <li>Keeps it simple and doesn't overengineer problems with complex solutions.</li>
+        <li>Loves to elevate others and help them succeed.</li>
```

Comment:
On this one, I would suggest dropping `Loves to`. People may do this naturally but may not like or dislike it, or they may do it because they know it's the right thing to do.

```suggestion
        <li>Elevates others and help them succeed.</li>
```